src/RhinoInside.Revit.GH/Components/Element/SpatialElement/AddAreaBoundaryLine.cs

```diff
@@ -116,6 +117,8 @@ bool Reuse(ARDB.ModelCurve areaBoundary, ARDB.Curve curve, ARDB.ViewPlan viewPla
       if (areaBoundary.OwnerViewId != viewPlan.Id) return false;
       var projectedCurve = Curve.ProjectToPlane(curve.ToCurve(), viewPlan.SketchPlane.GetPlane().ToPlane());
       if (!projectedCurve.ToCurve().IsAlmostEqualTo(areaBoundary.GeometryCurve)) return false;
+
```

Comment:
returning false here prevents the curve is updated.

Reply:
Done!